client/src/components/Charts/OrgPieChart.js

```diff
@@ -98,7 +98,7 @@ export class OrgPieChart extends Component {
 		const { classes } = this.props;
 		return (
 			<div className={classes.container}>
-				<PieChart width={485} height={290} className={classes.chart}>
+				<PieChart width={250} height={290} className={classes.chart}>
 					<Legend align="right" height={10} />
```

Comment:
Please remove the height from Legend, it is cropping out the details below piechart

Reply:
Removed the height parameter in the recent commit